### DISCUSSION TITLE
Maintainer

### DISCUSSION THREAD

**Discussion by MehlGibsSohn** [Hartz-Arbeit/LohntDat#25] [Informationen]
Damit ich hier nicht alleine dastehe, habe ich mich entschieden die 4, die bereits pull requests erstellt haben, als Maintainer einzustufen.

- @ByteDream
- @cr4yfish
- @jpv-os
- @timokae

**Comment by bytedream** [comment]
Es gibt ein Projekt für dieses Repo, was allerdings auf Privat gestellt ist und nur von einem Organization Admin auf Öffentlich gestellt werden kann. Das müsste mal gemacht werden @MehlGibsSohn.

Projekt (aktuell nur einsehbar für Owner und Maintainer) -> https://github.com/orgs/Hartz-Arbeit/projects/1

**Comment by bytedream** [comment]
Die Organization hat jetzt verschiedene Teams, aktuell für allgemeines Coding, Frontend mit React und UI/UX.
Das hilft z.B. bei PRs, anstatt dann von jeder einzelnen Person ein review zu requesten, kann man einfach das Team pingen.
Ich hoffe jeder Organization Member kann sich selbstständig zu Teams hinzufügen, hab damit noch nie gearbeitet.

Teams (nur einsehbar für Member) -> https://github.com/orgs/Hartz-Arbeit/teams

**Reply by cr4yfish** [reply]
Jo das joinen funktioniert. Aber das muss anscheinend ein bestehendes Team Mitglied approven.

**Comment by cr4yfish** [comment]
I move dieses Issue mal zu den Discussions, das macht mehr Sinn denke ich.
